common/abstractions/services/app.ts

```diff
@@ -28,33 +28,33 @@ export const EmailSettings: Record<Environments, EmailSettingsType> = {
     get production() {
         return {
             projectName: 'Companion kit',
-            sendgridTemplateId: 'd-047698cabf8846c985bda62b50850f2d',
-            fromAddress: 'Companion kit Team <your email>',
-            adminEmail: '<your email>',
+            sendgridTemplateId: 'd-5554ed3a2e7940c39daf251b4625895c',
```

Comment:
Note to self. Should change these to use the environment variables as well, when that is complete, even though they're not necessarily sensitive.